docs/admin/accessRights.rst

```diff
@@ -139,11 +139,38 @@ Programming Exercises
 +--------------------------------+------------+--------+--------------------+
 | Cleanup                        |      ✔     |        |                    |
 +--------------------------------+------------+--------+--------------------+
-| Repository Access              |    R + W   |  R + W |          R         |
-+--------------------------------+------------+--------+--------------------+
 | Build Plan Access              |    R + W   |  R + W |          R         |
 +--------------------------------+------------+--------+--------------------+
 
+Programming Exercises - Repository Access
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Action Types:
+    **READ** (fetch, pull), **WRITE** (push), **RESET** (reset to origin head)
+
+Access to all repositories of a course:
+
++------------------------------+---------------------+----------------+
+| Admin / Instructor / Editor  | Teaching Assistant  | Student        |
++------------------------------+---------------------+----------------+
+| *READ* , *WRITE* , *RESET*   | *READ*              | restricted     |
++------------------------------+---------------------+----------------+
+
+Access for Teaching Assistants and Students on their **own** repositories:
+
++--------------------------------------------------------------+------------------------------+------------+
+|                                                              | Teaching Assistant           | Student    |
++--------------------------------------------------------------+------------------------------+------------+
+| Before start                                                 | *READ*, *WRITE*, *RESET*     | restricted |
++--------------------------------------------------------------+------------------------------+------------+
+| After start but before due date and not locked               | *READ*, *WRITE*, *RESET*                  |
++--------------------------------------------------------------+-------------------------------------------+
+| After due date and practice mode enabled                     | *READ*, *WRITE*, *RESET*                  |
```

Comment:
The practice mode is a different repository.
```suggestion
```

Reply:
But isn't this still part of the endpoint then? So shouldn't this still be part of the documentation? It might be another repo but it's still their repo, isn't it? So I would still say "Practice Mode | *READ*, *WRITE*, *RESET*"

@Strohgelaender @mateusmm01

Reply:
From my understanding (since the practice mode is its own repository) it should never be possible for students to push into their own repository after the deadline.

Reply:
Yes, but not what I'm talking about:
This table is not about the "graded" repository (as tutors don't have a graded repository) but about the personal repository.
The practice mode repo is anothe repo but it's still their own repo. Every student et al can have up to two personal repos. One graded and one for practice. And there are different permissions on both.

That's how I understand it.

Reply:
Good point, can you clarify that the table is the "graded personal participation" (I think we can call it like this even for TAs). And then add a small sentence below indicating that practice repositories allow pushes after the due date.